sandevistan-server-hardening/CIS_Ubuntu_22.04_LTS_Benchmark_v1.0.0
# 1.5.2 ensure prelink is not installed

#### Summary 
  * *Description*: Implements 1.5.2 ensure prelink is not installed
  * *Issue*: https://github.com/sandevistan-server-hardening/CIS_Ubuntu_22.04_LTS_Benchmark_v1.0.0/issues/32
  * test environment: shanecfast/sandevistan-ubuntu-22.04-developer-level-0:latest
  
  
   
#### Manual testing
  * *Context*: prelink is a program that modifies ELF shared libraries and ELF dynamically linked
binaries in such a way that the time needed for the dynamic linker to perform relocations
at startup significantly decreases.
  * *How to test*: Run the audit.sh and test.sh files on a fresh Ubuntu 22.04 environment and see all the tests pass
  
------------------------

  
  Quote of the PR:
 "Learn as if you were to live forever.” - Mahatma Gandhi
